src/components/bc-modal.ts

```diff
@@ -22,15 +23,30 @@ export class Modal extends withTwind()(BitcoinConnectElement) {
       ></div>
       <div
         class="transition-all p-4 pt-6 pb-8 rounded-2xl shadow-2xl flex justify-center items-center w-full bg-white dark:bg-black max-w-md max-sm:rounded-b-none
-        animate-fade-in"
+        animate-fade-in ${this.isClosing
+          ? 'animate-slide-down'
+          : 'animate-slide-up'} sm:animate-none"
       >
         <slot @onclose=${this._handleClose}></slot>
       </div>
     </div>`;
   }
 
   private _handleClose = () => {
-    closeModal();
+    this.isClosing = true;
```

Comment:
Unfortunately this does not work so well and will not be executed if the modal is closed programmatically (see `closeModal` in `api.ts`). I think it's best to revert this change and only have an open animation for now until we find a more elegant way to do it.

There used to be a closing animation but it created a lot of complexity and it was removed in [this commit](https://github.com/getAlby/bitcoin-connect/commit/207033e5500312a5c340831b9fe9f9ef04b2593a)

there are some other issues in this modal too - like the backdrop disappears instantly. But I think we should just not do the closing animation for now.

Reply:
@rolznz, Sure I will remove the closing animation. :-)